src/Autofac/Core/Resolving/Pipeline/ResolveRequestContextBase.cs

```diff
@@ -59,7 +59,7 @@ public abstract class ResolveRequestContextBase : IComponentContext
             Parameters = request.Parameters;
             PhaseReached = PipelinePhase.ResolveRequestStart;
             Tracer = tracer;
-            TracingEnabled = tracer is object;
+            TracingEnabled = tracer is { };
```

Comment:
I feel like `is { }` is semantically more confusing than `is object`, so I'd  prefer to keep the original.

That syntax is really intended for checking for specific properties on the type, rather than just a null check.

Reply:
Okay, reverted that change in both places.